Kernel/Graphics/VirtIOGPU/GraphicsAdapter.cpp

```diff
@@ -101,20 +104,20 @@ ErrorOr<void> VirtIOGraphicsAdapter::attach_physical_range_to_framebuffer(VirtIO
 
     // 1. Create BUFFER using VIRTIO_GPU_CMD_RESOURCE_CREATE_2D
     if (buffer.resource_id.value() != 0)
```

Comment:
Nit: While you are here, maybe put this in braces? IIRC that's a style guide thing.

Reply:
Also, technically, we should be deleting the resource if any of the subsequent steps fail as well (to avoid leaking the resource number), I think, but for now maybe a fixme would be better.